_std/defines/component_defines/component_defines_global.dm

```diff
@@ -36,3 +36,21 @@
 /// ⠀⠀⠀⠀⠀⠀⠀⠈⠛⠻⠿⠿⠿⠿⠋⠁⠀⠀⠀⠀
 #define COMSIG_GLOBAL_SUSSY_PHRASE "sussy"
 #define COMSIG_GLOBAL_UNCOOL_PHRASE "uncool_word"
+
+// ---- Trader Shuttle Movement ----
+
+	/// When a trader is arriving/departing at the 'left' trading area on-station
+	#define COMSIG_TRADER_LEFT "trader_left"
```

Comment:
`COMSIG_TRADER_LEFT` feel like it's going to cause confusion eventually between left (direction) and left (departed). Not a big deal but disambiguating this early wouldn't hurt